src/RangeSliderWithInputs/RangeSliderWithInputs.tsx

```diff
@@ -3,6 +3,12 @@ import { styled } from '@mui/material/styles';
 import React from 'react';
 import { RangeSliderWithInputsProps } from './RangeSliderWithInputs.types';
 
+type InpuEvent = {
```

Comment:
typo here I think @voj should this  be  `InputEvent` ?

Reply:
Good catch!